test/integration/tasks.test.js

```diff
@@ -261,7 +260,7 @@ describe('Tasks', function () {
     it('Should return 200 when username is valid', function (done) {
       chai
         .request(app)
-        .get(`/tasks/${appOwner.username}`)
+        .get(`/tasks/${appOwner.username}?status=active`)
```

Comment:
Can we also have a test for the case when status is not sent?

Reply:
@swarajpure Is it ok if I add these test under [this](https://github.com/Real-Dev-Squad/website-backend/issues/381) task as under this task I have to write almost all test cases again, which will revamp whole code of tasks integration test.

Reply:
@swarajpure Will do this change under [this](https://github.com/Real-Dev-Squad/website-backend/issues/381) task.